mediation-old/src/main/java/com/wso2telco/dep/mediator/RequestExecutor.java

```diff
@@ -1670,11 +1663,7 @@ private Operator getOperatorByName(String operator) throws OperatorServiceExcept
 
     private String modifyEndpoint(String sendingAdd, String operator, MessageContext context){
 
-        FileReader fileReader = new FileReader();
-        String file = CarbonUtils.getCarbonConfigDirPath() + File.separator
-                      + FileNames.MEDIATOR_CONF_FILE.getFileName();
-        Map<String, String> mediatorConfMap = fileReader.readPropertyFile(file);
-        String esbEndpoint = mediatorConfMap.get("esbEndpoint");
+	    String esbEndpoint = ConfigFileReader.getInstance().getMediatorConfigMap().get("esbEndpoint");
```

Comment:
Formatting: indentation

Reply:
Please check the source code. this might be a presentaion problem in git hub. 
Correctly formatted in the source.